models/dashboard_bilhetagem_madonna/transacao_madonna.sql

```diff
@@ -0,0 +1,38 @@
+{{
+  config(
+    alias="transacao"
+  )
+}}
+
+WITH transacao_deduplicada AS (
+    SELECT 
+        * EXCEPT(rn)
+    FROM
+    (
+        SELECT
+            *,
+            ROW_NUMBER() OVER (PARTITION BY id ORDER BY timestamp_captura DESC) AS rn
+        FROM
+            {{ ref('staging_transacao') }}
+        where
+            data >= "{{ var('run_date') }}"
```

Comment:
alterar para DATE(data) BETWEEN DATE("{{ var('run_date') }}") and DATE_ADD(DATE("{{ var('run_date') }}"), INTERVAL 1 DAY)